src/current/cockroachcloud/costs.md

```diff
@@ -24,9 +28,9 @@ This table summarizes key details about how costs are calculated for each plan t
-| [Data transfer](#data-transfer) | Usage-based via Request Units. | Usage-based; rates vary per cloud provider and data transfer type.<br><br>Currently not charged during [Usage-based billing metrics Preview](#usage-based-billing-metrics-in-preview). | Usage-based; rates vary per cloud provider and data transfer type.<br><br>Currently not charged during [Usage-based billing metrics Preview](#usage-based-billing-metrics-in-preview). |
-| [Change Data Capture](#change-data-capture-changefeeds) | Usage-based via Request Units. | Usage-based, per GiB-Hour watched across all changefeeds.<br><br>Currently not charged during [Usage-based billing metrics Preview](#usage-based-billing-metrics-in-preview). | Usage-based, per GiB-Hour watched across all changefeeds. Rates vary depending on whether security add-on is enabled.<br><br>Currently not charged during [Usage-based billing metrics Preview](#usage-based-billing-metrics-in-preview). |
+| [Backups](#backups) | Usage-based via Request Units. | Usage-based.<br><br>[Managed Backups]({% link cockroachcloud/managed-backups.md %}) storage rates vary per cloud provider, region, and backup frequency.<br><br>[Self-Managed Backups]({% link cockroachcloud/take-and-restore-self-managed-backups.md %}) fee incurred per GiB transferred to your own object storage. | Usage-based.<br><br>[Managed Backups]({% link cockroachcloud/managed-backups.md %}) storage rates vary per cloud provider, region,  backup frequency, and whether security add-on is enabled.<br><br>[Self-Managed Backups]({% link cockroachcloud/take-and-restore-self-managed-backups.md %}) fee incurred per GiB transferred to your own object storage. Rate varies depending on whether security add-on is enabled. |
+| [Data transfer](#data-transfer) | Usage-based via Request Units. | Usage-based; rates vary per cloud provider and data transfer type. | Usage-based; rates vary per cloud provider and data transfer type. |
+| [Change Data Capture](#change-data-capture-changefeeds) | Usage-based via Request Units. | Usage-based, per GiB-Hour watched across all changefeeds. | Usage-based, per GiB-Hour watched across all changefeeds. Rates vary depending on whether security add-on is enabled. |
```

Comment:
What do we mean by watched? Is is the total amount of data in the tables or the total amount of data that matches selectors defined for the changefeed? I suspect it's the total amount of data that might be streamed (thus data matching selectors, but it's not entirely clear).